TAO/tao/Strategies/DIOP_Acceptor.cpp

```diff
@@ -1038,6 +1071,20 @@ TAO_DIOP_Acceptor::parse_options (const char *str)
                                  ACE_TEXT ("endpoint priorities no longer supported.\n")),
                                 -1);
             }
+          else if (name == "portspan")
+            {
+              int range = static_cast <int> (ACE_OS::atoi (value.c_str ()));
```

Comment:
could be const

Reply:
as ACE_OS::atoi returns int a cast is not necessary here

Reply:
Good to know. Done per your suggestion below.